### PR TITLE
bug: cache older CNI and CNS binaries to sync with RP

### PR DESCRIPTION
**Pull request by ganeshkumarashok**
**What type of PR is this?**
/kind bug 

**What this PR does / why we need it**:
Syncing CNI and CNS versions with those needed by AKS-RP, to prevent egress to ACS mirror for these components during provisioning. 

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -50,6 +53,9 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/cni-dropgz:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
+        "v0.0.4.1",
+        "v0.0.11",
+        "v0.0.15",
```
Comment (cameronmeissner) [inline vhdbuilder/packer/components.json:58]: should be fine to not update prefetch optimizations for now I suppose, will probably need to come back to it after we finalize plans with the image builder folks tho

**Review by rbtr — APPROVED** [review]

**Review by wedaly — APPROVED** [review]

File: vhdbuilder/packer/generate-windows-vhd-configuration.ps1 [hunk 2]
```diff
@@ -112,7 +112,10 @@ $global:imagesToPull += @(
     "mcr.microsoft.com/containernetworking/azure-cns:v1.4.51",
     "mcr.microsoft.com/containernetworking/azure-cns:v1.4.52",
     "mcr.microsoft.com/containernetworking/azure-cns:v1.5.15",
```
Comment (rbtr) [inline vhdbuilder/packer/generate-windows-vhd-configuration.ps1:114]:
```suggestion
    "mcr.microsoft.com/containernetworking/azure-cns:v1.4.52",
```

File: vhdbuilder/packer/components.json [hunk 3]
```diff
@@ -368,17 +369,17 @@
       "downloadLocation": "/opt/cni/downloads",
       "downloadURL": "https://acs-mirror.azureedge.net/azure-cni/v*/binaries",
       "versions": [
-        "1.4.51",
-        "1.5.15"
+        "1.4.43.1",
+        "1.5.11",
```
Comment (rbtr) [inline vhdbuilder/packer/components.json:373]:
```suggestion
        "1.5.11"
```
dangling `,` breaks the JSON

File: vhdbuilder/packer/components.json [hunk 4]
```diff
@@ -368,17 +369,17 @@
...
     },
     {
       "fileName": "azure-vnet-cni-swift-linux-amd64-v*",
       "downloadLocation": "/opt/cni/downloads",
       "downloadURL": "https://acs-mirror.azureedge.net/azure-cni/v*/binaries",
       "versions": [
-        "1.4.51",
-        "1.5.15"
+        "1.5.11",
+        "1.4.43.1",
```
Comment (rbtr) [inline vhdbuilder/packer/components.json:382]:
```suggestion
        "1.4.43.1"
```
dangling `,` breaks the JSON

**Review by rbtr — COMMENTED** [review]
commit my previous suggestions resulted in invalid JSON

**Review by anujmaheshwari1 — APPROVED** [review]

**Pull request merged**